### DISCUSSION TITLE
Square artifacts in color data on jpeg outputs

### DISCUSSION THREAD

**Discussion by jrdiver** [SixLabors/ImageSharp#1547] [Q&A] [answered]
I was messing with the Resize sample just trying to figure out the library, but I keep getting some weird 
block artifacts when the quality is set to less then a jpg quality of 91 on ImageSharp's NuGet package 1.0.3-Alpha.0.35

I reverted it back to the latest stable - 1.0.2 and the issue went away.  Code used is basicly the sample besides removing grayscale and changing output to jpg with a quality setting.

`
           System.IO.Directory.CreateDirectory("output");

            using (Image image = Image.Load("fb.jpg"))
            {
                image.Mutate(x => x.Resize(image.Width / 2, image.Height / 2));

            image.SaveAsJpeg("output/Photo.jpg", new JpegEncoder { Quality = 75 }); // Automatic encoder selected based on extension.
            }
`

### System Configuration
Windows 10/Visual Studio 2019/NetCore3.1

![fb](https://user-images.githubusercontent.com/12983591/107839316-9fd17080-6d70-11eb-9ea6-1620e0f2f986.jpg)
![Photo](https://user-images.githubusercontent.com/12983591/107839317-a1029d80-6d70-11eb-9d30-26e2d153fe38.jpg)

**Comment by JimBobSquarePants** [comment]
@jrdiver This would have been better raised as a discussion. Please in future use the supplied template. It's very important that you do so. 

Please, with all the required information, also supply comparison output images so that we do not have to spend time replicating your process before we can even determine whether there is an issue.

**Reply by jrdiver** [reply]
First picture is just from the samples, second is my profile picture, here's the original.  I did try several other pictures and got similar results, reverting back to the latest stable "solved" the issue, but if there does seem to be something going on with the latest prerelease.

![DSC02610](https://user-images.githubusercontent.com/12983591/107842777-5d695d00-6d8b-11eb-8f1a-38860928be37.JPG)

**Reply by JimBobSquarePants** [reply]
Can you please label the images so I can tell which output image is from which version.

**Reply by JimBobSquarePants** [reply]
Ok.... I've done some investigating and there's definitely an issue introduced with #1517 Thanks for bringing this to our attention.

**Comment by antonfirsov** [comment]
I can confirm this by running `JpegEncoderTests` and checking the output.

This is a serious bug in JpegEncoder, and also a big reliability issue with `JpegEncoderTests`, will open an issue soon.

@jrdiver thanks for reporting!

**Reply by JimBobSquarePants** [reply]
@antonfirsov  I've opened #1549 if you want to add to that.

**Reply by antonfirsov** [reply]
Yeah, you were faster 😄

**Comment by antonfirsov** [comment, marked as answer]
Fixed by #1551 in `1.0.3-alpha.0.37`:
https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp

**Reply by jrdiver** [reply]
The new build seems to be working as expected with Jpg's now.  Thanks!
